packages/tomls/src/core/configs/exchanges.toml

```diff
@@ -33,4 +33,19 @@ var.lotteryPotExchangeId.arg = 0
 depends = [
     "invoke.exchange_pass_lottery_pot_mint_owner",
     "invoke.core_configure_protocol",
+    "invoke.core_register_exchange_passive_pool",
```

Comment:
why do we need this additional dependence, think it worked without it before ?

Reply:
it worked before, and it will work now as well. this dependency is good for the future, in case we want to redeploy the entire protocol from scratch -- it will keep them in order (pool 1, lottery 2, etc)